src/library_wasmfs_node.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright 2022 The Emscripten Authors
+ * SPDX-License-Identifier: MIT
+ */
+
+mergeInto(LibraryManager.library, {
+  $wasmfs$node$isWindows: !!process.platform.match(/^win/),
+
+  $wasmfs$node$convertNodeCode__deps: ['$ERRNO_CODES'],
+  $wasmfs$node$convertNodeCode: function(e) {
+    var code = e.code;
+#if ASSERTIONS
+    assert(code in ERRNO_CODES, 'unexpected node error code: ' + code + ' (' + e + ')');
+#endif
+    return ERRNO_CODES[code];
+  },
+
+  $wasmfs$node$lstat__deps: ['$wasmfs$node$isWindows', '$wasmfs$node$convertNodeCode'],
+  $wasmfs$node$lstat: function(path) {
```

Comment:
This is a lot of `$`s, not sure we can afford them all :smile: 

Seriously though it seems a little different than other code. Usually we do JS style names for JS things, like `$wasmfsNodeLstat` perhaps?